tests/test_core/test_sequence.py

```diff
@@ -2655,3 +2657,48 @@ def test_seqview_slice_propagates_seqid():
 
     copied_sliced_sv = sliced_sv.copy(sliced=True)
     assert copied_sliced_sv.seqid == "seq1"
+
+
+@pytest.mark.parametrize("cls", (Aligned, Sequence, SeqView, ArraySequence, str, bytes))
+def test_coerce_to_seqview(cls):
+    seq = "ACGGTGGGAC"
```

Comment:
change sample data to
```python
seq = "AC--GGTGGGAC"
```
and the previous version will fail for the `Aligned` class